src/main/java/homework/lectures/lecture6/animal/Animal.java

```diff
@@ -0,0 +1,102 @@
+package homework.lectures.lecture6.animal;
+
+import java.util.Objects;
+
+public class Animal {
```

Comment:
You need to make this class abstract because right now you can create instance of this class but its not properly in real life

Reply:
Thx, I fix it!